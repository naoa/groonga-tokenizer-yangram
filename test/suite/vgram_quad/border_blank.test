plugin_register tokenizers/yangram

table_create vgram_words TABLE_PAT_KEY ShortText
load --table vgram_words
[
{"_key": "材は"}
]

tokenize TokenYaVgramQuadSplitDigit '過材 ウム 。' --normalizer NormalizerAuto --mode GET
