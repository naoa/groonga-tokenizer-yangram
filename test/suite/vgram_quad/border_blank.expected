plugin_register tokenizers/yangram
[[0,0.0,0.0],true]
table_create vgram_words TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
load --table vgram_words
[
{"_key": "材は"}
]
[[0,0.0,0.0],1]
tokenize TokenYaVgramQuadSplitDigit '過材 ウム 。' --normalizer NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "過材",
      "position": 0,
      "force_prefix": true
    },
    {
      "value": "ウム",
      "position": 2,
      "force_prefix": true
    },
    {
      "value": "。",
      "position": 4,
      "force_prefix": false
    }
  ]
]
